content/recipes/typeorm-migration.md

```diff
@@ -0,0 +1,112 @@
+### TypeORM migration
+
+#### Objectives
+Generate and use migrations instead of syncing database. In dev and prod.
+
+#### Pre-requisites
+TypeORM installed: https://docs.nestjs.com/techniques/database
+
+#### Install
+
+**src/app.module.ts**
+```typescript
+import { DynamicModule, Module } from '@nestjs/common';
+import { AppController } from './app.controller';
+import { AppService } from './app.service';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import * as ormconfig from './ormconfig';
+
+export function DatabaseOrmModule(): DynamicModule {
```

Comment:
Would this be better as a function or a `const` like
```ts
export const DatabaseModule: DynamicModule = TypeOrmModule.forRoot(ormConfig);
```
This way it stays in the same vein that _usually_ raw functions are not used for the `imports` array.